br/backup-and-restore-storages.md

```diff
@@ -175,7 +176,7 @@ It is recommended that you configure access to S3 using either of the following
 </div>
 <div label="GCS" value="gcs">
 
-You can configure the account used to access GCS by specifying the access key. If you specify the `credentials-file` parameter, the authentication is performed using the specified `credentials-file`. Besides specifying the key in the URL, the following methods are also supported:
+You can configure the account used to access GCS by specifying the access key. If you specify the `credentials-file` parameter, the authentication is performed using the specified `credentials-file`. Besides specifying the key in the URI, the following methods are also supported:
```

Comment:
Does the Chinese doc version need the same update?

Reply:
yes